apps/daimo-mobile/src/logic/chain.ts

```diff
@@ -1,33 +1,45 @@
 import { createContext, useEffect, useState } from "react";
 import {
   Address,
+  Log,
   PublicClient,
   createPublicClient,
+  getAbiItem,
   getContract,
   http,
 } from "viem";
-import { goerli, baseGoerli } from "viem/chains";
+import { baseGoerli, goerli } from "viem/chains";
 import { erc20ABI } from "wagmi";
 
-import { generateKeypair } from "./crypto";
 import { Account, useAccount } from "./account";
+import { assert } from "./assert";
+import { generateKeypair } from "./crypto";
+import { notify } from "./notify";
 import { trpc } from "./trpc";
 import { check } from "./validation";
-import { assert } from "./assert";
 
 export const chainConfig = {
   testnet: true,
   l1: goerli,
   l2: baseGoerli,
-  coinContract: "0x1B85deDe8178E18CdE599B4C9d913534553C3dBf" as Address,
 };
 
+export const coin = {
+  address: "0x1B85deDe8178E18CdE599B4C9d913534553C3dBf" as Address,
+  decimals: 6,
```

Comment:
nit and dont need to do here but I was thinking we can make @daimo/contracts generate and export some of the metadata of the erc20 contract separately like `address`/`decimals`/`name`/`symbol` etc. so we don't have to hardcode or fetch in-app. Will also make simpler to swap it out in future for us (and others who fork)